src/jlgen.jl

```diff
@@ -318,7 +319,8 @@ else
     get_method_table_view(world::UInt, mt::MTType) = OverlayMethodTable(world, mt)
 end
 
-struct GPUInterpreter <: CC.AbstractInterpreter
+abstract type AbstractGPUInterpreter <: CC.AbstractInterpreter end
+struct GPUInterpreter <: AbstractGPUInterpreter
     world::UInt
```

Comment:
Is this still needed? I don't see any other interpreters used here.

Reply:
So Enzyme has it's own Interpreter. And the idea was that we could subtype AbstractGPUInterpreter and then reuse the code here. I still have to proof that out though.